source/pipeWith.js

```diff
@@ -10,7 +10,8 @@ import identity from './identity';
  * Performs left-to-right function composition using transforming function. The leftmost function may have
  * any arity; the remaining functions must be unary.
  *
- * **Note:** The result of pipeWith is not automatically curried.
+ * **Note:** The result of pipeWith is not automatically curried. Transforming function is not used on the
```

Comment:
I would rather love to find some alternative to `"leftmost" and "rightmost" (here and in `pipe`, `compose` and `composeWith`.)  While it's probably clear enough most of the time, it's not really accurate, in that I often lay out `pipe` functions vertically.

If you have any suggestions, please let me know.

Reply:
Updated for `composeWith`. How about simply `first argument` and `last argument`?

Reply:
Thank you.

Yes, I think that `first argument` and `last argument` would be better.  

It's not ideal, but anything I can come up with that's really accurate sounds like a legalism.

Reply:
Updated. I've updated `compose`, `composeP`, `pipe`, `pipeP` as well. :)